test/test_wrapper.py

```diff
@@ -320,6 +320,7 @@ def test_image_format_constructor():
         assert not hasattr(iform, "__dict__")
 
 
+@pytest.mark.skipif(not hasattr(cl, "DeviceTopologyAmd"), reason="not AMD procesor")
```

Comment:
This isn't correct: What symbols are by the ICD loader has nothing to do with what platforms are available.

Reply:
I'm not familiar with this, but during the test on apple `DeviceTopologyAmd` was missed. Finally, I disable most of the tests on macOS (also this) because of segfaults. 

So I may remove this line. What Is your opinion?

Reply:
Reverting this proposed change is fine with me.